### DISCUSSION TITLE
Issue when tryin to get return period extremes: all cores taken at 100%, and never ending

### DISCUSSION THREAD

**Discussion by maucollu** [georgebv/pyextremes#53] [Q&A] [answered]
Hi, I'm pretty new to using pyextremes.
I tried to run the [quickstart](https://georgebv.github.io/pyextremes/quickstart/) example, and I can successfully reproduce the steps until:
`model.fit_model()`

Unfortunately, when I try to launch:
`summary = model.get_summary(     return_period=[1, 2, 5, 10, 25, 50, 100, 250, 500, 1000],     alpha=0.95,     n_samples=1000, )`

the code is running forever, and is taking ALL my 12 cores at 100%.

I'm using miniconda, I run a script in Spyder with all the commands. Python 3.10.8. Windows 11 Pro

I am not sure, but it seems that the problem is in how pyextremes use parallel cores?

Thanks a lot for your help

**Comment by georgebv** [comment, marked as answer]
I recommend reading this discussion https://github.com/georgebv/pyextremes/discussions/20

**Reply by maucollu** [reply]
Thanks a lot, @georgebv  it worked - apologies if you had to answer the same questions over and over again. I looked already at the linky you sent before asking, but was not python-savvy enough to recognise that it was the solution!
